### DISCUSSION TITLE
Suggestions for Container Grid (for the future)

### DISCUSSION THREAD

**Discussion by Michaelgimii** [elementor/elementor#21175] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Hi elementor team!

Since container grid is in Elementor roadmap and we are close to it, last night I was checking this feature in different site builders. What caught my attention was the Grid in the webflow website builder. In webflow, container grid is very simple and professional.
So I wanted to mention two of them and I hope it will be implemented in Elementor. Because it is really wonderful and user friendly. Also, I will post the link to review it by webflow, maybe viewing it will bring more interesting ideas for you in Elementor.

### Describe the solution you'd like

1- Ability to add/delete rows and columns using one click.

https://user-images.githubusercontent.com/88034902/215252748-dc2999c7-3699-4281-9632-9bd801b2fded.mp4

2- Ability to merge grids using drag. fantastic!

https://user-images.githubusercontent.com/88034902/215253589-425bb8c3-d5f3-450d-9e70-e0c322d939ad.mp4

I hope these things will help you in the development of container grid.

The link to the full review of the Grid container in webflow : https://youtu.be/1ViSR5fSDwg

**Comment by nicholaszein** [comment]
This request was mentioned in the following issues and discussions:

- #24188

**Comment by sara-kariv** [comment]
+1
